src/plugin/context/mod.rs

```diff
@@ -772,21 +772,20 @@ impl RapierContext {
     }
 
     /// Finds all entities of all the colliders with an Aabb intersecting the given Aabb.
-    #[cfg(not(feature = "headless"))]
     pub fn colliders_with_aabb_intersecting_aabb(
         &self,
-        aabb: bevy::render::primitives::Aabb,
+        aabb: bevy::math::bounding::Aabb3d,
         mut callback: impl FnMut(Entity) -> bool,
```

Comment:
add the same for aabb2d

Reply:
Yes, I think it would be cleaner if the 2d version required `Aabb2d` instead.